src/style.css

```diff
@@ -0,0 +1,61 @@
+@import url("https://fonts.googleapis.com/css?family=Roboto:400,500&display=swap&subset=cyrillic");
+
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  height: 60px;
+}
+
+.logo {
+  margin: 10px auto 10px 50px;
+}
+
+.navbar-list {
+  margin: 0 50px 0 0;
+  padding: 0;
+  display: flex;
+  text-decoration: none;
+  list-style: none;
+  height: 60px;
+}
+
+a {
+  text-decoration: none;
+}
+
+.nav_link {
+  font-family: inherit;
+  line-height: 60px;
+  text-transform: uppercase;
+  color: #000;
+  margin-left: 20px;
+}
+
+.nav_link:hover,
+.is-active {
+  color: #00acdc;
+}
+
+.is-active {
+  margin-left: 0;
```

Comment:
`is-active` must only change color and add bottom line
```suggestion
```
![image](https://user-images.githubusercontent.com/44294945/64270317-04fe2100-cf44-11e9-8a42-f36e9357d24b.png)